web-common/src/features/dashboards/rows-viewer-accordion/RowsViewerAccordion.svelte

```diff
@@ -0,0 +1,113 @@
+<script lang="ts">
+  import CaretUpIcon from "@rilldata/web-common/components/icons/CaretUpIcon.svelte";
+  import RowsViewer from "../rows-viewer/RowsViewer.svelte";
+  import CaretDownIcon from "@rilldata/web-common/components/icons/CaretDownIcon.svelte";
+  import { createQueryServiceMetricsViewTotals } from "@rilldata/web-common/runtime-client";
+  import { runtime } from "@rilldata/web-common/runtime-client/runtime-store";
+  import { formatCompactInteger } from "@rilldata/web-common/lib/formatters";
+  import { useDashboardStore } from "../dashboard-stores";
+  import { useModelHasTimeSeries } from "../selectors";
+  export let metricViewName: string;
+  let isOpen = false;
+  const toggle = () => {
+    isOpen = !isOpen;
+  };
+
+  $: totalsQuery = createQueryServiceMetricsViewTotals(
+    $runtime.instanceId,
+    metricViewName,
+    {
+      measureNames: ["count"],
+      inlineMeasures: [
+        {
+          name: "count",
+          expression: "count(*)",
+        },
+      ],
+    },
+    {
+      query: {
+        queryKey: ["dashboardAllRowsCt"],
```

Comment:
Why the custom `queryKey`? If it's necessary, mind adding a code comment to explain?

Reply:
It should not be necessary, the queryKey generated by the custom hook should be enough to uniquely identify the query.

However, in practice I've discovered that this query frequently fails to put any data into the cache when using the autogenerated key. Instead, the query runs, the data is fetched from the server, but the query status remains at "loading" and the fetch status says "idle".

I will send you a video of this behavior. The only way I found to fix it for now is to provide a custom query key. I could use your help investigating the underlying issue; it smells like a svelte-query bug.

Reply:
I added a comment about this